bundles/org.openhab.binding.easee/src/main/resources/OH-INF/i18n/easee.properties

```
@@ -0,0 +1,166 @@
+# binding
+
+name = Easee Binding
+description = This is the binding for Easee Wallboxes.
```

Prefixes here removed by mistake:
```suggestion
binding.easee.name = Easee Binding
binding.easee.description = This is the binding for Easee Wallboxes.
```

After fixing this, you could regenerate this file as also some new strings were added in XML.